torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4311,6 +4311,26 @@ def sample_inputs_narrow_copy(op_info, device, dtype, requires_grad, **kwargs):
         yield SampleInput(tensor, args=args)
 
 
+def sample_inputs_view_copy(op_info, device, dtype, requires_grad, **kwargs):
+    shapes = (
+        ((S, S), [S * S, 1]),
+        ((S, S, S), [S * S, S]),
+        ((S, S, S), [S * S * S]),
+    )
+
+    for shape, new_shape in shapes:
+        tensor = make_tensor(shape, dtype=dtype, device=device, low=None, high=None,
+                             requires_grad=requires_grad)
+        yield SampleInput(tensor, new_shape)
+
+
+def error_inputs_view_copy(op_info, device):
+    tensor = make_tensor([S, S], dtype=torch.float32, device=device)
+    # Size is too big
+    s0 = SampleInput(tensor, [S * S * 10])
+    yield ErrorInput(s0, error_regex=r"shape '\[250\]' is invalid for input of size 25", error_type=RuntimeError)
+
```

Comment:
```python
>>> torch.view_copy(x, (-1, 3, -1))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: only one dimension can be inferred
```

I think this would be a nice case to add.

Reply:
Also, probably in a follow-up PR, it would be nice to use this function to add error inputs for `reshape` and `view`.